tests/drivers/can/api/src/main.c

```diff
@@ -657,7 +657,7 @@ static void test_set_bitrate(void)
  * This test case must be run before sending/receiving test cases as it allows
  * these test cases to send/receive their own frames.
  */
-static void test_set_loopback(void)
+ZTEST_USER(can_api_test, test_set_loopback)
```

Comment:
This test is intended to be run before any test involving can communication.
So this should be run as setup function or guaranteed otherwise.

Reply:
Agreed.

Reply:
yes, change it to be part of setup function.